exe/rubydown

```diff
@@ -62,6 +62,9 @@ class EvalDoc < Kramdown::Document
     case
     when eval_result.respond_to?(:to_html)
       Kramdown::Document.new(eval_result.to_html, input: :html).root
+    when eval_result.instance_of?(File) && eval_result.path[-4,4] == ".png"
+      img_b64 = Base64.encode64(File.binread(eval_result))
```

Comment:
`File.binread` accepts file path string.
If `eval_result` is IO, use `eval_result.read` after open with `'rb'` (read binary mode).
Or pass `eval_result` as file path string.